exporter/awsxrayexporter/internal/translator/segment.go

```diff
@@ -22,6 +23,11 @@ import (
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/coreinternal/traceutil"
 )
 
+var allowRandomTimestamp = featuregate.GlobalRegistry().MustRegister(
+	"exporter.awsxray.allowrandomtimestamp",
+	featuregate.StageAlpha,
+	featuregate.WithRegisterDescription("Remove XRay's timestamp validation on first 32 bits of trace ID"))
```

Comment:
I think there is a `featuregate.FromVersion()` option. Can you please add that with collector version `v0.84.0`.

Reply:
Added tests and `featuregate.WithRegisterFromVersion("v0.84.0")`